docs-user/guide-getting-started.md

```diff
@@ -41,7 +41,7 @@ The same settings are available in the devtools panel.
 ![A screenshot of the Devtools performance panel](images/getting-started-devtools-panel.png)
 
 !> As explained above, when opening the Devtools toolbox there's an
-overhead due to the other panels. We try that it stays minimal but you need to
+overhead. We try to maintain a minimal overhead, but you need to
```

Comment:
I think I'd like to keep the idea that the other panels in the Devtools toolbox bring this overhead. Do you have a suggestion?

Reply:
:+1: sounds good. Let me know if the update is better, or if it needs more work.